### ISSUE TITLE
Project unavailable in WrapDB

### ISSUE THREAD

**Issue by JPTIZ** [mesonbuild/check#2]
I tried to access this project in WrapDB (https://wrapdb.mesonbuild.com/check) but seems to be missing, since the page states "No such project" and using the `meson wrap` tool the following happens:

```console
$ meson wrap info check

ERROR: WrapDB connection failed to https://wrapdb.mesonbuild.com/v1/projects/check with error HTTP Error 404: NOT FOUND
```

Maybe with #1 fixed, merged and if a release is made it'll work.

**Comment by legeana** [comment]
https://github.com/mesonbuild/check/tree/0.10.0 I don't think there is any code there. So since there are no releases there is nothing to display. wrapdb is not great yet at displaying the error messages :(
Would you be up to fix https://github.com/mesonbuild/check/pull/1#issuecomment-589430917? Then we can merge it.

**Comment by JPTIZ** [comment]
I think I can give a try, yes. Let me take a look :)

**Comment by legeana** [comment]
Fixed by #3. Thank you @JPTIZ for help!
